.teamcity/Gradle_Check/configurations/FunctionalTest.kt

```diff
@@ -7,23 +7,29 @@ import model.Stage
 import model.TestCoverage
 import model.TestType
 
-class FunctionalTest(model: CIBuildModel, testCoverage: TestCoverage, subProject: String = "", stage: Stage) : BaseGradleBuildType(model, stage = stage, init = {
-    uuid = testCoverage.asConfigurationId(model, subProject)
+class FunctionalTest(model: CIBuildModel, testCoverage: TestCoverage, subProjects: List<String> = listOf(), stage: Stage, buildTypeName: String = "") : BaseGradleBuildType(model, stage = stage, init = {
+    val coverageName = if (subProjects.size > 1)  buildTypeName else subProjects.joinToString("")
+    uuid = testCoverage.asConfigurationId(model, coverageName)
```

Comment:
Will it exceeds the limitation of TC? (80 characters I remember for configuration id)